lib/babe/verify.go

```diff
@@ -50,7 +51,7 @@ func NewVerificationManager(blockState BlockState, epochState EpochState) (*Veri
 		return nil, ErrNilBlockState
 	}
 
-	if epochState == nil {
+	if reflect.ValueOf(epochState).IsNil() {
```

Comment:
Just curious, when does this become a typed nil?

Reply:
I'm not sure, and I don't recall why this was changed.

Reply:
this isn't changed on development, we shouldn't change it unless it's required.